### PR TITLE
[FSPE-5175] Added -xs size in toggle switch component

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSPE-5175

@janibolkvadze

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://880f19ea2c9b22ead11909f1e954c255cf915a26-chi-pr.enterprise.ctl.io" target="_blank"> https://880f19ea2c9b22ead11909f1e954c255cf915a26-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1207/ for more details.

File: src/custom-elements/src/components/toggle-switch/toggle-switch.tsx [hunk 1]
```diff
@@ -26,6 +26,11 @@ export class ToggleSwitch {
    */
   @Prop({ reflect: true, mutable: true }) checked = false;
 
+  /**
+   * OPTIONAL. Size of the label. {xs,sm}.
+   */
+  @Prop({ reflect: true }) size: string;
```
Comment (janibolkvadze) [inline src/custom-elements/src/components/toggle-switch/toggle-switch.tsx:32]:
```suggestion
  @Prop({ reflect: true }) size?: string;
```
Let's add an interrogation symbol here to define the prop as optional.
We should do the same for all the optional props of other web components as well

**Comment by janibolkvadze** [comment]
👍 👍

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://38c7b4e937aa77744e7e32a05718cce5f7dace77-chi-pr.enterprise.ctl.io" target="_blank"> https://38c7b4e937aa77744e7e32a05718cce5f7dace77-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1210/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f23ad3a339d86b9db5470fed34114d1a8ee34db3-chi-pr.enterprise.ctl.io" target="_blank"> https://f23ad3a339d86b9db5470fed34114d1a8ee34db3-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1216/ for more details.

**Comment by janibolkvadze** [comment]
LGTM

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://63872766f0f50c5a180adca4cc41c938cd28b1ab-chi-pr.enterprise.ctl.io" target="_blank"> https://63872766f0f50c5a180adca4cc41c938cd28b1ab-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1220/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-fkqd38, https://ops.elasticbox.com/services/instances/i-d28rbi, https://ops.elasticbox.com/services/instances/i-ojwjpk, https://ops.elasticbox.com/services/instances/i-t936b2

**Pull request merged**